src/pages/rooms.tsx

```diff
@@ -0,0 +1,80 @@
+import { NextPage } from "next";
+import React, { useEffect, useState } from "react";
+import { useStores } from "@/stores/context";
+import { observer } from "mobx-react-lite";
+import { GET_ROOMS_FAILED, ROOM_CREATE_FAILED } from "@/constants/roomMessage";
+
+const ManageRooms: NextPage = observer(() => {
```

Comment:
방 목록 페이지라서 단순하게 아래처럼 수정하는게 어떨까여? RoomListStore와 이름 형식을 맞추는 의미도 있어보이네요.
```suggestion
const RoomList: NextPage = observer(() => {
```